src/Agent.Worker/Build/TfsVCSourceProvider.cs

```diff
@@ -96,7 +96,7 @@ public sealed class TfsVCSourceProvider : SourceProvider, ISourceProvider
                 StringUtil.ConvertToBoolean(endpoint.Data[WellKnownEndpointData.Clean], defaultValue: false);
             if (tf.Features.HasFlag(TfsVCFeatures.Scorch) || !clean)
             {
-                existingTFWorkspace = MatchExactWorkspace(
+                existingTFWorkspace = WorkspaceUtil.MatchExactWorkspace(
```

Comment:
I could have left this method in the same class and made it public and static (for unit tests), but it felt cleaner to move it to a separate class which is public and static.

Nothing much changed about this particular method other than that.